pwiz_tools/Skyline/TestTutorial/ExistingExperimentsTutorialTest.cs

```diff
@@ -641,6 +651,22 @@ private static void NormalizeGraphToHeavy()
             SkylineWindow.UpdatePeakAreaGraph();
         }
 
+        private void VerifyRdotPLabels(string[] replicates, string[] rdotps)

```

Comment:
This is not going to work right when you run the test in French where numbers are formatted differently. You are going to need to have a list of doubles, and call the method double.ToString(formatString, CultureInfo.CurrentCulture).